indexer/feature_meta.cpp

```diff
@@ -41,6 +41,16 @@ string Metadata::GetWikiURL() const
   return "https://" + v.substr(0, colon) + kBaseWikiUrl + v.substr(colon + 1);
 }
 
+string Metadata::GetWikimediaCommonsURL() const
+{
+  const string base_url = "https://commons.wikimedia.org/wiki/";
```

Comment:
Hardcode this URL below in the return. Otherwise, it's a wasteful operation when v.empty() returns true.

Reply:
done